src/main/java/uk/gov/hmcts/ccd/appinsights/CallbackTelemetryThreadContext.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.ccd.appinsights;
+
+public class CallbackTelemetryThreadContext {
```

Comment:
can we add a Javadoc explaining the usage of this class

Reply:
Maybe let's call it `CallbackTelemetryThreadContext` so that in the future we can use it outside of callbacks as well?

Reply:
sorry didn't get that